README.md

```diff
@@ -182,8 +182,8 @@ sources: "BBC,NPR,FOX,CNBC,CNN"
 The way summarization works can be configured as follows:
 
 ```yaml
-# Hide or show summary (if not specified defaults to True). When set to False, it hides the summary and prevents the call to the summarization API.
-enable_summary: True
+# Switches between summary mode and search results mode (if not specified defaults to True). When set to True, a summary is shown along with references used in the summary. When set to False, only search results are shown and no calls made to the summarization API.
+summary_mode: True
```

Comment:
Are you saying we should add something like a toggle?

Reply:
No, I'm just suggesting that instead of `summary_mode: True` we offer a configuration option of `ux: "summary"`, and instead of `summary_mode: False` we offer a configuration option of `ux: "search"`.